README.rst

```diff
@@ -296,20 +295,13 @@ Path to an executable script that will be called after the registration is compl
 Compiling openwisp-config
 -------------------------
 
-There are 4 variants of *openwisp-config*:
-
-- **openwisp-config-openssl**: depends on *ca-certificates* and *libopenssl*
-- **openwisp-config-mbedtls**: depends on *ca-certificates* and *libmbedtls*
-- **openwisp-config-wolfssl**: depends on *ca-certificates* and *libwolfssl*
-- **openwisp-config-nossl**: doesn't depend on any SSL library and doesn't install trusted CA certificates
-
-The following procedure illustrates how to compile all the *openwisp-config* variants and their dependencies:
+The following procedure illustrates how to compile all the *openwisp-config* and their dependencies:
```

Comment:
Better:

`The following procedure illustrates how to compile *openwisp-config* and its dependencies:`